pkg/apis/v1alpha1/suite_test.go

```diff
@@ -115,4 +120,96 @@ var _ = Describe("Validation", func() {
 			Expect(ant.Validate(ctx)).To(Not(Succeed()))
 		})
 	})
+	var _ = Describe("AWSNodeTemplate Annotation", func() {
```

Comment:
nit: I'd name it Hash
```suggestion
	var _ = Describe("AWSNodeTemplate Hash", func() {
```